MediaPlayer/MediaControls.js

```diff
@@ -98,6 +98,7 @@ const MediaControls = kind({
 		 * `true` when the media is paused.
 		 *
 		 * @type {Boolean}
+		 * @default true
```

Comment:
MediaControls looks private component. right?

Reply:
Yes. `MediaPlayer` is public.